materialize-bigquery/sqlgen.go

```diff
@@ -120,32 +116,11 @@ var bqDialect = func() sql.Dialect {
 			return "?"
 		}),
 		TypeMapper:             mapper,
-		ColumnValidator:        columnValidator,
 		MaxColumnCharLength:    300,
 		CaseInsensitiveColumns: true,
 	}
 }()
 
-// bignumericCompatible allows either integers or numbers, since we used to create number columns as
```

Comment:
As noted here, we used to create `type: number` columns as bignumeric but have been creating them as float64 for quite a long time. I'm removing the ability to validate pre-existing bignumeric columns for `type: number` fields, unless there is a user-defined DDL set. There are probably some existing tasks that will need to have this DDL set for as part of the migration.